src/services/expenses.services.js

```diff
@@ -0,0 +1,47 @@
+const { getExpensesFilterQuerry } = require('../utils/expenses');
+
+const { Expense } = require('../models/Expense.model');
+
+const getExpenses = async (categories, userId, from, to) => {
+  const where = getExpensesFilterQuerry(categories, userId, from, to);
```

Comment:
There is a typo in the function name `getExpensesFilterQuerry`. It should be `getExpensesFilterQuery` to match the correct spelling of 'query'. Ensure the function definition in `../utils/expenses` is also updated accordingly.